# A single waypoint in a mission plan

geometry_msgs/Pose pose
float32 dwell_time
string[] actions
int32 priority
